recipe/meta.yaml

```diff
@@ -28,7 +29,6 @@ requirements:
   host:
     - libarrow
   run:
-    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
```

Comment:
We need to check this change.

Reply:
That is ok, see https://conda-forge.org/news/2024/03/24/stdlib-migration/ . Probably however we can remove the `run` section as it is empty.